libs/damap/src/lib/components/dmp/people/people.component.html

```diff
@@ -253,7 +273,9 @@
             <div class="mat-card-container">
               <div class="mat-card-left">
                 <div>
-                  <mat-icon class="mat-icon-style">person</mat-icon>
+                  <mat-icon class="mat-icon-style outline"
+                    >person_outline</mat-icon
+                  >
```

Comment:
Can you please revert this? it doesnt look as good on the white/gray background and some of those icons at another place are still black - so we have an inconsistency .
```suggestion
                  <mat-icon class="mat-icon-style">person</mat-icon>
```